shardingsphere-underlying/shardingsphere-rewrite/shardingsphere-rewrite-engine/src/main/java/org/apache/shardingsphere/underlying/rewrite/engine/RouteSQLRewriteEngine.java

```diff
@@ -62,7 +62,10 @@ public RouteSQLRewriteResult rewrite(final SQLRewriteContext sqlRewriteContext,
         int count = 0;
```

Comment:
Above this line, there is return early style code,   Should exclude when parameters is empty and onduplicateparameter is not empty.